web/src/app/store/Classification-state.ts

```diff
@@ -0,0 +1,65 @@
+import { Action, Selector, State, StateContext } from '@ngxs/store';
```

Comment:
This state is still one huge file.
Lets talk about how we can handle states which will get bigger.

Reply:
https://itnext.io/ngxs-thoughts-patterns-architecture-and-best-practices-c991c42618d9

=> extract selectors + actions